deployment/blackduck-alert.env

```diff
@@ -21,15 +26,14 @@ ALERT_ENCRYPTION_GLOBAL_SALT=
 
 # -- BlackDuck Provider Settings
 
+#ALERT_PROVIDER_BLACKDUCK_PROVIDER_COMMON_CONFIG_NAME=Default Config
+#ALERT_PROVIDER_BLACKDUCK_PROVIDER_COMMON_CONFIG_ENABLED=true
```

Comment:
Same as above, why isnt this `DEFAULT_CONFIG_ENABLED` ?

Reply:
see the answer above.